Arcade/TNT Floor/map.xml

```diff
@@ -0,0 +1,46 @@
+<map proto="1.4.0">
+<name>TNT Floor</name>
+<version>1.0.0</version>
+<objective>Shoot out the TNT under the other team to win!</objective>
+<gamemode>arcade</gamemode>
+<authors>
+    <authors> <author uuid="d1873b18-8194-429b-b6d5-eb7eb897071d"/> <!-- AVVERSION --> </authors>
+</authors>
+<contributors>
+    <contributor uuid="a9f11454-beac-4cc3-a138-1780802c8794" contribution="XML"/> <!-- Crazy_ --> </contributors>
+</contributors>
+<teams>
+    <team id="red" color="dark red" max="3">Red</team>
+    <team id="blue" color="blue" max="3">Blue</team>
+</teams>
+<kits>
+    <kit id="play">
+        <item slot="0" enchantment="arrow infinite;flame">bow</item>
+        <item slot="28">arrow</item>
+        <gamemode>adventure</gamemode>
+        <gamemode>blitz</gamemode>

```

Comment:
It's used on warzone for stat tracking, aka the arcade game mode disables stats.

Reply:
The issue @CrazyisCreeps is that it's currently in the <kits> module. It needs to be defined on its own - which looks like it's already been done.

Reply:
My mistake. I used the `<gamemode>` tag instead of `<game-mode>` for adventure mode in the kit, but GitHub wasn't allowing me to edit this request. In another request, I added a suggestion that put the blitz tag in the correct place.